client/state/reader/teams/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External Dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_TEAMS_REQUEST,
+	READER_TEAMS_RECEIVE,
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+// TODO add in error handling
```

Comment:
@blowery  / @bluefuton: were we previously doing any error handling w.r.t failed teams requests?